pkg/auth/inject.go

```diff
@@ -84,3 +87,11 @@ func (m DependencyMap) Provide(dependencyName string, r *http.Request) interface
 		return nil
 	}
 }
+
+func sensitiveLoggerValues(r *http.Request) []string {
+	tConfig := config.GetTenantConfig(r)
+	return []string{
+		tConfig.APIKey,
+		tConfig.MasterKey,
+	}
+}
```

Comment:
Missed this, this should belong to core, not auth.

Reply:
@rickmak updated. thanks.